save-cloud-common/src/commonMain/kotlin/org/cqfn/save/entities/UserDto.kt

```diff
@@ -0,0 +1,18 @@
+package org.cqfn.save.entities
+
+import org.cqfn.save.domain.Role
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property name
+ * @property source
+ * @property projects
+ * @property source where the user identity is coming from, e.g. "github"
+ */
+@Serializable
+data class UserDto(
```

Comment:
`UserInfo` is used at frontend while UserDto is used to pass data from backend to frontend and bakcwards. I thought it is a good idea to split those structures. Do you think it might be better to merge them?

Reply:
I think yes, if some new requests will require additional information, we could create new data classes, which contain UserInfo. I don't see an obvious reason to keep two classes now

Reply:
Done